src/models/student.js

```diff
@@ -8,72 +8,72 @@ const subject = mongoose.Schema({
   },
 });
 
-const student = mongoose.Schema({
-  username: {
-    type: String,
-    required: true,
-    unique: true,
-  },
-  firstName: {
-    type: String,
-    required: true,
-  },
-  lastName: {
-    type: String,
-    required: true,
-  },
-  isSignUpWithGoogle: {
-    type: Boolean,
-    default: false,
-  },
-  passwordHash: {
-    type: String,
-    required() {
-      return this.isSignUpWithGoogle === false;
+const student = mongoose.Schema(
+  {
+    username: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    isSignUpWithGoogle: {
+      type: Boolean,
+      default: false,
+    },
+    passwordHash: {
+      type: String,
+      required: () => this.isSignUpWithGoogle === false,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    isTutor: {
+      type: Boolean,
+      default: false,
+    },
+    avatar: {
+      type: String,
+    },
+    subjects: {
+      type: [subject],
+    },
+    questions: {
+      type: [mongoose.Schema.Types.ObjectId],
+      ref: 'question',
+    },
+    messagingChannels: {
+      type: [mongoose.Schema.Types.ObjectId],
+      ref: 'MessagingChannel',
     },
   },
-  email: {
-    type: String,
-    required: true,
-    unique: true,
-  },
-  isTutor: {
-    type: Boolean,
-    default: false,
-  },
-  avatar: {
-    type: String,
-  },
-  subjects: {
-    type: [subject],
-  },
-  questions: {
-    type: [mongoose.Schema.Types.ObjectId],
-    ref: 'question',
-  },
-  messagingChannels: {
-    type: [mongoose.Schema.Types.ObjectId],
-    ref: 'MessagingChannel',
-  },
-},
-{
-  toObject: {
-    virtuals: true,
-  },
-  toJSON: {
-    virtuals: true,
-  },
-});
+  {
+    toObject: {
```

Comment:
Remove this line, while resolving the conflicts you will see that they added a new line that works similar to line 79 and it also sets virtuals to true.
To keep things organized remove `toObject` from here and add it similarly to line 79 (you will find it implemented while resolving the conflicts).